core/controllers/tasks.py

```diff
@@ -75,6 +75,34 @@ def post(self):
 class SuggestionEmailHandler(base.BaseHandler):
     """Handler task of sending email of suggestion."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'exploration_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'thread_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'exploration': {
+                'schema': {
+                    'type': 'basestring'
+                },
+            },
+            'suggestion': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
+        }
+    }
+
     @acl_decorators.can_perform_tasks_in_taskqueue
     def post(self):
         payload = json.loads(self.request.body)
```

Comment:
I think this shoudl also be replaced `self.normalized_payload`. @vojtechjelinek  can you PTAL?

Reply:
Right!